drivers/ata/ahci.c

```diff
@@ -633,7 +633,7 @@ static struct pci_driver ahci_pci_driver = {
 
 static int ahciem_enable = 1;
 module_param(ahciem_enable, int, 0644);
-MODULE_PARM_DESC(ahciem_enable, "Emulate SES enclosure (0 = disabled)");
+MODULE_PARM_DESC(ahciem_enable, "Emulate SES enclosure (0 = disabled, 2 = forced)");
```

Comment:
On FreeBSD I've used -1 as default (not set), 0 as disabled, 1 as enabled.  I wonder whether 2 is used in some other code to be consistent.